tests/workers/test_weave_scorer.py

```diff
@@ -0,0 +1,117 @@
+from datetime import datetime
+from unittest.mock import Mock, patch
+
+import pytest
+from src.weave_scorer import (
+    get_active_monitors,
+)
+
+from weave.flow.monitor import Monitor
+from weave.flow.scorer import Scorer
+from weave.trace_server import trace_server_interface as tsi
+from weave.trace_server.refs_internal import InternalObjectRef
+from weave.trace_server.trace_server_interface import ObjQueryRes, ObjSchema
+
+
+@pytest.fixture
+def mock_trace_server():
+    server = Mock()
+    server.objs_query = Mock()
+    server.call_start = Mock()
+    server.call_end = Mock()
+    return server
+
+
+@pytest.fixture
+def mock_monitor_object_res():
+    return ObjQueryRes(
+        objs=[
+            ObjSchema(
+                project_id="test-project",
+                object_id="test-monitor-1",
+                created_at=datetime.now(),
+                version_index=1,
+                is_latest=1,
+                kind="object",
+                base_object_class="Monitor",
+                val={
+                    "name": "test-monitor-1",
+                    "description": "Test Monitor 1",
+                    "sampling_rate": 1.0,
+                    "scorers": ["scorer1", "scorer2"],
```

Comment:
maybe this is just example data, but it seems sus to me that scorers can be non-refs